composer.json

```diff
@@ -11,7 +11,8 @@
 		}
 	],
 	"require": {
-		"neos/media": ">3"
+		"neos/media": ">3",
```

Comment:
could be adjusted to >4 at least? or will there be a version 5 of the package to make that one compatible with Flow 6 as well? if so we wouldn't need the flow requirement here